package.json

```diff
@@ -16,6 +16,8 @@
     "migration:undo": "sequelize db:migrate:undo:all",
     "migrate": "sequelize db:migrate",
     "seed": "sequelize db:seed:all",
+    "admin:create": "nodemon --exec babel-node src/database/superAdmin.js",
```

Comment:
No need to use `nodemon --exec` it runs continuously and does not exit. You can remove it and it will still work

Reply:
sure.